modules/betweenBidAdapter.js

```diff
@@ -74,9 +74,14 @@ export const spec = {
         }
       }
 
-      requests.push({method: 'GET', url: 'https://ads.betweendigital.com/adjson', data: params})
+      requests.push({data: params})
     })
-    return requests;
+    return {
+      method: 'POST',
+      url: 'https://ads.betweendigital.com/adjson?t=prebid',
```

Comment:
Just a suggestion, but might be a good idea to set this as a const at the top of your adapter? Like `STR_ENDPOINT`?